webview-ui/src/Kaito/state.ts

```diff
@@ -45,3 +55,36 @@ export const vscode = getWebviewMessageContext<"kaito">({
     generateWorkspaceRequest: null,
     deployWorkspace: null,
 });
+
+export const vscode2 = getWebviewMessageContext<"kaitoModels">({
+    generateCRDRequest: null,
+    deployKaitoRequest: null,
+    workspaceExistsRequest: null,
+    updateStateRequest: null,
+    resetStateRequest: null,
+    cancelRequest: null,
+});
+
+export const stateUpdater2: WebviewStateUpdater<"kaitoModels", EventDef, DeploymentState> = {
```

Comment:
Can you explain why we would need stateUpdater2 and can't reuse previous stateUpdater ? just for the documentation purpose.

Reply:
Gotcha, I'll make sure to add documentation for this in further PR 👍

Reply:
I hope this has been refactored in future PR's ?

Reply:
📓 Nice, seems like @tejhan  mentioned that he will take care of most of this in PR here post this merge: https://github.com/Azure/vscode-aks-tools/pull/1003 , leaving this as fyi comment.